frmts/georaster/georaster_wrapper.cpp

```diff
@@ -543,6 +558,77 @@ GeoRasterWrapper *GeoRasterWrapper::Open(const char *pszStringId, bool bUpdate)
     return poGRW;
 }
 
+static bool ValidateInsertExpression(const CPLString &sInsertStatement)
+{
+    bool bInQuotes = false;
+    const size_t nStrLength = sInsertStatement.length();
+
+    for (size_t nPos = 0; nPos < nStrLength; ++nPos)
+    {
+        const char cCurrentChar = sInsertStatement[nPos];
+
+        if (cCurrentChar == '\'')
+        {
+            if (bInQuotes)
+            {
+                if (nPos + 1 < nStrLength && sInsertStatement[nPos + 1] == '\'')
+                {
+                    ++nPos;
+                }
+                else
+                {
+                    bInQuotes = false;
+                }
+            }
+            else
+            {
+                bInQuotes = true;
+            }
+        }
+        else if (!bInQuotes)
+        {
+            if (cCurrentChar == ';')
+            {
+                return false;
+            }
+            else if (nPos < nStrLength - 1)
+            {
+                const char cNextChar = sInsertStatement[nPos + 1];
+                const bool bIsInvalid =
+                    (cCurrentChar == '-' && cNextChar == '-') ||
+                    (cCurrentChar == '/' && cNextChar == '/') ||
+                    (cCurrentChar == '/' && cNextChar == '*') ||
+                    (cCurrentChar == '*' && cNextChar == '/');
+
+                if (bIsInvalid)
+                {
+                    return false;
+                }
+            }
+        }
+    }
+    // Quotes must be properly closed, if not, this variable will be true
+    // thus having an unclosed string
+    return !bInQuotes;
+}
+
+static bool ValidateDescriptionExpression(const CPLString &sInsertStatement)
+{
+    const char *rgpszInvalidChars[] = {";", "--", "/*", "*/", "//"};
+    const size_t nInvalidCharsSize = 5;
+
+    for (size_t nPos = 0; nPos < nInvalidCharsSize; ++nPos)
+    {
+        const char *pszInvalidChar = rgpszInvalidChars[nPos];
```

Comment:
can be simplified as:
```suggestion
    const char *rgpszInvalidChars[] = {";", "--", "/*", "*/", "//"};
    for (const char *pszInvalidChar: rgpszInvalidChars)
    {
```